tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -270,6 +270,25 @@ public TracerSettings(IConfigurationSource source)
             {
                 HttpClientExcludedUrlSubstrings = TrimSplitString(urlSubstringSkips.ToUpperInvariant(), commaSeparator);
             }
+
+            DbmPropagationLevel dbmPropagationValue;
+            DbmPropagationMode = Enum.TryParse(source?.GetString(ConfigurationKeys.DbmPropagationMode), true, out dbmPropagationValue) ?
```

Comment:
I think we can change the type of the `DbmPropagationMode` property to an enum so other parts of the code are comparing against an enum